app/src/main/java/edu/grinnell/appdev/grinnelldirectory/activities/SearchResultsActivity.java

```diff
@@ -43,4 +53,31 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return true;
     }
+
+
+    private void loadPersons() {
+        // get multiple persons
+        String json = loadJSONFromAsset("dummyData.json");
+        Persons persons = new Gson().fromJson(json, Persons.class);
+        System.out.println(persons.getPersons().get(0).getFirstName());
+
+        // return our persons object
+        this.mPeopleList = persons.getPersons();
+    }
+
+    public String loadJSONFromAsset(String fileName) {
+        String json = null;
+        try {
+            InputStream is = this.getAssets().open(fileName);
```

Comment:
This variable should be declared outside the try-catch so that it can be closed in the finally block.

Reply:
Closing needs to be in the try block because it can throw an exception.